test/native/perfEventsLinuxTest.cpp

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifdef __linux__
+
+#include "callTraceStorage.h"
+#include "perfEvents.h"
+#include "testRunner.hpp"
+#include <linux/perf_event.h>
+#include <stdio.h>
+
+#define ASSERT_EVENT_TYPE(event_type, name_, type_, default_interval_, config_, config1_, config2_, counter_arg_) \
```

Comment:
I see you've changed indentation to 2 spaces. Async-profiler uses 4 spaces - this is one of the differences from HotSpot coding style; sorry I haven't mentioned that in advance.